src/RequestHandler.h

```diff
@@ -1,17 +1,24 @@
 #pragma once
 
-#include "ConnectKafka.h"
+#include "FlatbuffersTranslator.h"
 #include "RequestHandlerInterface.h"
-#include <librdkafka/rdkafkacpp.h>
+#include <spdlog/logger.h>
+#include <spdlog/spdlog.h>
 
 class RequestHandler : public RequestHandlerInterface {
 private:
   std::unique_ptr<ConnectKafkaInterface> KafkaConnection;
+  void consumePartitions(std::pair<std::string, bool> MessageAndEOF,
+                         int &EOFPartitionCounter,
+                         FlatbuffersTranslator &FlatBuffers);
+  std::shared_ptr<spdlog::logger> Logger;
```

Comment:
`Logger` isn't used, best to add this only when you know it is required, always assume you aren't going to need it.